ansible-dryad/roles/dryad_app/templates/bash_profile.j2

```diff
@@ -1,11 +1,20 @@
-export PATH="~/bin:{{ dryad.tomcat_home }}/bin:$PATH"
+export PATH="$HOME/.local/bin:$HOME/bin:{{ dryad.tomcat_home }}/bin:$HOME/dryad-utils:$PATH"
 
+# AWS settings:
+export AWS_ACCESS_KEY_ID={{ dryad.aws.accessKey }}
+export AWS_SECRET_ACCESS_KEY={{ dryad.aws.secretKey }}
+
+# Postgres settings:
 # Requires .pgpass
 export PGHOST={{ dryad.db.host }}
 export PGPORT={{ dryad.db.port }}
 export PGUSER={{ dryad.db.user }}
 export PGDATABASE={{ dryad.db.name }}
 
+# aliases and vars for dryad-utils and 
+export DRYAD_DB_HOST=$PGHOST
+alias psql_dryad='psql --host $DRYAD_DB_HOST -U dryad_app -d dryad_repo'
```

Comment:
Looks like this takes the place of bin/postgres_client.sh  Is there any reason to retain both of them?

Reply:
I didn't even notice that bin/postgres_client existed! So I guess not.

Reply:
Which do you prefer?

Reply:
I think it's better as a script, so other scripts can call it.